libs/db-browser/src/main/java/com/oceanbase/tools/dbbrowser/model/DBTableConstraint.java

```diff
@@ -26,7 +26,7 @@
 
 @Data
 @EqualsAndHashCode(
-        exclude = {"name", "warning", "schemaName", "tableName", "createTime", "updateTime", "ordinalPosition",
+        exclude = {"name", "warning", "schemaName", "owner", "tableName", "createTime", "updateTime", "ordinalPosition",
```

Comment:
Because `DBTableConstraintEditor` uses `Objects.equals(oldConstraint, newConstraint)` when judging whether two constraints are equal. I think `owner` is just like `schemaName` should not be included when judging whether two constraints are equal.